src/main/java/com/jdbc/demo/domain/psql/FreightTransport.java

```diff
@@ -62,14 +62,14 @@
     private Client client;
 
     @ManyToMany
-    @JoinTable(name="FreightTransportVehicles", joinColumns = { @JoinColumn(name="id_FreightTransport") },
-            inverseJoinColumns = { @JoinColumn(name="id_Vehicle") })
-    private List<Vehicle> vehicles = new ArrayList<>();
-
-    @ManyToMany
-    @JoinTable(name="FreightTransportDrivers", joinColumns = { @JoinColumn(name="id_FreightTransport") },
-            inverseJoinColumns = { @JoinColumn(name="id_Driver") })
-    private List<Driver> drivers = new ArrayList<>();
+	@JoinTable(name = "FreightTransportData", joinColumns = {
+			@JoinColumn(name = "id_FreightTransport") }, inverseJoinColumns = { 
+					@JoinColumn(name = "id_Vehicle") })
+	private List<Vehicle> vehicles = new ArrayList<>();
+	@ManyToMany
+	@JoinTable(name = "FreightTransportData", joinColumns = {
+			@JoinColumn(name = "id_FreightTransport") }, inverseJoinColumns = { @JoinColumn(name = "id_Driver") })
+	private List<Driver> drivers = new ArrayList<>();
 
```

Comment:
Please add appropriate methods for adding and removing vehicles and drivers pairs from `FreighTransportData` table.